### ISSUE TITLE
Fix Makefile "test-package-files" task

### ISSUE THREAD

**Issue by matthewmorgan** [exercism/ecmascript#303]
We made a well-intentioned change to our Makefile that tests to see if each exercise has a `package.json` file that matches the global `xecmascript/package.json` file, and fails the build if not.

However, there is no easy way for a contributor to update all the `package.json` files when there is a change.  It is critical that these files are all updated, and we want this to happen before PRs are submitted, so the build fails if it is not done.  

Two possible solutions:
1. Have the Makefile check its environment.  If `Travis=true` then we're in CI and should always SKIP copying the `package.json` files from global to each exercise.
2. Add a task to the Makefile that would let contributors copy the package.json file to all exercises, EG `make copy-package-to-exercises`.  

Either approach probably warrants a small update to the contributor docs.

Success Criteria:
- Builds fail on Travis if `package.json` files are not the same in each exercise as the global.
- Contributors have a simple way to make sure their PRs are ready for CI testing.
- Documentation is updated.

**Comment by rchavarria** [comment]
I prefer taking the second way, adding the new `copy-package-to-exercises` task.

I'm wondering if we need a new task to be able to *build* the project, to run tests independently of copying `package.json` or checking them.

Let's say we have these tasks:

- `make test`: it should just run tests, no `package.json` checking, no `package.json` copying
- `make copy-package-to-exercises`: easy to understand
- `make build`: it could check `package.json` equality **and** run tests.

**Comment by rchavarria** [comment]
@exercism/ecmascripts - Is this still relevant after #336?

**Comment by matthewmorgan** [comment]
@rchavarria #336 is not exactly this implementation, and wasn't intended to be.  I still think this idea has value, but what do you (and others) think?

**Comment by rchavarria** [comment]
:+1: Me too
